libvast/src/chunk.cpp

```diff
@@ -75,11 +75,15 @@ caf::expected<chunk_ptr> chunk::mmap(const std::filesystem::path& filename,
     return caf::make_error(ec::filesystem_error,
                            fmt::format("failed to open file {}", filename));
   auto map = ::mmap(nullptr, size, PROT_READ, MAP_SHARED, fd, offset);
+  auto mmap_errno = errno;
   ::close(fd);
   if (map == MAP_FAILED)
     return caf::make_error(ec::filesystem_error,
-                           fmt::format("failed to mmap file {}", filename));
-  auto deleter = [=]() noexcept { ::munmap(map, size); };
+                           fmt::format("failed to mmap file {}: {}", filename,
+                                       strerror(mmap_errno)));
```

Comment:
We could use `std::strerror` if you prefer. In general, I prefer using the standard library equivalents rather than the POSIX calls directly.